jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/message/Asn1DecodeMessageJSON.java

```diff
@@ -45,19 +47,22 @@ public Asn1DecodeMessageJSON(OdeProperties odeProps) {
 
 	@Override
 	protected Object process(String consumedData) {
-		OdeData odeData = null;
-		OdeMsgPayload payload = null;
```

Comment:
Let us share these two variables:  OdeData odeData = null; OdeMsgPayload payload = null; among different messages. Only move the OdeBsmMetadata/OdeLogMetadata metadata to local variables.